book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
```

Comment:
The `.create` method does return the library, so you can skip the assignment and go with this:
```suggestion
   return  Library.objects.create(
```

Reply:
Oh nice! That makes perfect sense!

Reply:
This is a really cool trick, coming from Ruby with implicit returns I like how you can get creative with them in Python.